server/src/test/java/keywhiz/service/daos/SecretControllerTest.java

```diff
@@ -79,112 +79,112 @@
 
   @Test
   public void createsSecretWithOwner() {
-    String owner = UUID.randomUUID().toString();
-    groupDAO.createGroup(owner, "creator", "description", ImmutableMap.of());
+    String ownerName = UUID.randomUUID().toString();
+    groupDAO.createGroup(ownerName, "creator", "description", ImmutableMap.of());
```

Comment:
I wonder if it makes sense to overload the `createGroup` method with just three args if the fourth `metadata` map is going to be empty often like this (or would this just be for testing?)

Reply:
Typically what I'll do for tests is have some helper method that takes in just the one or two parameters I care about and then defaults the others, but I just didn't do that here. Maybe there wasn't enough duplication to make it worthwhile (yet).